GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Athena.cfg

```diff
@@ -3,7 +3,7 @@
 
 //  Athena Launch Vehicle Family:  https://www.sprsa.org/sites/default/files/conference-presentation/Athena%20Launch%20Vehicle%20Family-Kehrl_PR.pdf
 //  Space Launch Report - Athena:  http://www.spacelaunchreport.com/athena.html
-//  Norbert Br�gge - Athena:       http://www.b14643.de/Spacerockets_2/United_States_2/Athena/Description/Frame.htm
+//  Norbert Br�gge - Athena:       http://www.b14643.de/Spacerockets_2/United_States_2/Athena/Description/Frame.htm
```

Comment:
Stomping of unicode.  Egg would murder you.

Reply:
Fixed. Hope VSCode hasn't messed up anything else